testscommon/goroutines/filter.go

```diff
@@ -13,6 +13,7 @@ var ignorable = []string{
 	"github.com/libp2p/go-libp2p-nat.(*NAT)",
 	"net._C2func_getaddrinfo",
 	"net.cgoLookupIP", //  for net.cgoLookupIP and net.cgoLookupIPCNAME
+	"github.com/libp2p/go-libp2p/p2p/host/peerstore/pstoremem.(*memoryAddrBook)", // TODO remove it after pubsub update, fixed here https://github.com/libp2p/go-libp2p-pubsub/pull/568
```

Comment:
How can we make sure we don't forget?

Reply:
forgot, of course.. removed as it was released